packages/coinstac-graphql/src/data/resolvers.js

```diff
@@ -123,15 +123,20 @@ const resolvers = {
           rethink.table('consortia').insert(
             args.consortium,
             { 
-              conflict: "replace",
+              conflict: "update",
               returnChanges: true,
             }
           )
           .run(connection)
+          .then((result) => {
+            //r.db('coinstac').table('users').filter({id: 'test'}).update({'permissions': {'consortia': {'some-new-id': {'write': true}}}})
+            let crazything = {};
+            crazything[result.changes[0].new_val.id] = {'write': true};
+            rethink.table('users').filter({id: 'test'}).update({'permissions': {'consortia': crazything}})
```

Comment:
😏 Should probably use something a bit more descriptive instead of `crazything`. You could actually use `consortia` as the name and then use [ES6 Property Value Shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer) since the property and the value would have the same name:

[(Another Example)](https://github.com/airbnb/javascript#es6-object-concise)

```
  let consortia = {};
  consortia[result.changes[0].new_val.id] = {'write': true};
  rethink.table('users').filter({id: 'test'}).update({'permissions': { consortia }})
```

Also, watch your spacing on objects. When defining an object, MRN uses the [AirBnB styleguide](https://github.com/airbnb/javascript), which [recommends padding when using curly braces](https://github.com/airbnb/javascript#whitespace--in-braces). The linter should generally warn you about this, and it will fail once we get testing running again so it's not a bad habit to get into now.